packages/input/__tests__/input.spec.ts

```diff
@@ -273,6 +273,31 @@ describe('Input.vue', () => {
       expect(input.selectionStart).toEqual(0)
       expect(input.selectionEnd).toEqual(testContent.length)
     })
+    test('method:resizeTextarea', async () => {
+      const testContent = 'TEXT:resizeTextarea'
+      const wrapper = _mount({
+        template: `<el-input  ref="textarea"  :autosize="{ minRows: 1, maxRows: 1 }" type="textarea" v-model="text" />`,
+        data() {
+          return {
+            text: testContent,
+          }
+        },
+      })
+      const _ref = wrapper.vm.$refs.textarea
+      await nextTick()
+      const originHeight = _ref.$el.clientHeight
+
+      _ref.autosize.minRows = 5
+      _ref.autosize.maxRows = 5
+
+      _ref.resizeTextarea()
+      // Textarea height will change
+      setTimeout(()=>{
+        const nowHeight = _ref.$el.clientHeight
+        expect(originHeight===nowHeight).toBe(false)
+      },1000)
```

Comment:
We've had a `sleep` function in `@element-plus/test-utils/sleep`, use that instead.
```suggestion
const nowHeight = _ref.$el.clientHeight
expect(originHeight).not.toEqual(nowHeight)
```

Reply:
i get it

Reply:
Thank you for being patient with my comments.